eyakimovich/01_HTML/Python001.html

```diff
@@ -0,0 +1,71 @@
+ <!DOCTYPE html>  
+ <HTML> 
+ <HEAD>
+ <META charset="utf-8">
+ <TITLE>Python</TITLE>
+ </HEAD>  
+ <BODY> 
+ 
+ <ul><H1>HOW-TO: Программа на Python, Часть 1</H1></ul>
+ 
+  <ul><h3><i>
```

Comment:
should not be here h3 and i as a first child inside ul. 
see slides, how looks the lists in html